lib/cli/handle-error.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+const path = require('path');
+const { inspect } = require('util');
+const isError = require('type/error/is');
+const isObject = require('type/object/is');
+const chalk = require('chalk');
+const isStandaloneExecutable = require('../utils/isStandaloneExecutable');
+const resolveLocalServerlessPath = require('./resolve-local-serverless-path');
+const slsVersion = require('./../../package').version;
+const sfeVersion = require('@serverless/enterprise-plugin/package.json').version;
+const { sdkVersion } = require('@serverless/enterprise-plugin');
+
+const userErrorNames = new Set(['ServerlessError', 'YAMLException']);
+const serverlessPath = path.resolve(__dirname, '../Serverless.js');
+
+const resolveExceptionMeta = (exception) => {
+  if (isError(exception)) {
+    return {
+      name: exception.name,
+      title: exception.name.replace(/([A-Z])/g, ' $1'),
+      stack: exception.stack,
+      message: exception.message,
+    };
+  }
+  return {
+    title: 'Exception',
+    message: inspect(exception),
+  };
+};
+
+const consoleLog = (message) => process.stdout.write(`${message}\n`);
+
+const writeMessage = (title, message) => {
```

Comment:
This is duplicated across two modules or I'm missing the difference between them?

Reply:
Good point, I didn't notice (linter simply told me it's still used in `Error.js` so I didn't touch it).

As I see this logic is also used in `logWarning`, and it's the reason it's also kept there.

I'm not sure is it worth to clean this up, as (1) `logWarning` and all other log functions will be removed with https://github.com/serverless/serverless/issues/1720 which is also on priority list, and (2) `handle-error.js` as introduced here is also likely to be improved  (in context of this PR we simply take this logic out of `Serverless` class context, so `Serverless` is not mangled with CLI specific logic)

It's likely that quite soon `writeMessage` will be gone from both locations (I also personally don't see it as part of `cli` utils), and investing time now to make a tested reusable util from it now, doesn't feel that justified.

Let me know

Reply:
If we'll be removing it soon I see zero harm in keeping in duplicated for this short period of time :+1: